docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
 sphinx-rtd-theme
-robotframework>=3.2.1
\ No newline at end of file
+robotframework>=3.2.2
```

Comment:
This is requirement only for building the docs. The reason is that we're using Robocop while building the docs (for example to get configurables) and when importing it, we need to also import rf. It can be 3.2.2 or even 4.0.0 since it only affects docs building

Reply:
Should I update it to 4.0.0?

Reply:
Yeah let's update it